tests/stdlib/t16967.nim

```diff
@@ -0,0 +1,16 @@
+import sugar
```

Comment:
move this to tsugar?

Reply:
tsugar is a mess, hopefully `push exportc` won't cause problems